dashboard-widgets.el

```diff
@@ -86,6 +86,10 @@ to the specified width, with aspect ratio preserved."
   (expand-file-name (concat dashboard-banners-directory "logo.png"))
   "Emacs banner image.")
 
+(defvar dashboard-banner-logo-text
```

Comment:
I don't know what I should use here because I saw `defcustom` and `defvar` everywhere between the user customize code and package code. So here I simply used `defvar`.

Reply:
Since this value should be customizable, I think it should be a `defcustom`.

Reply:
Resolved!